src/hw_isolation_event/hw_status_manager.cpp

```diff
@@ -106,6 +121,183 @@ std::pair<event::EventMsg, event::EventSeverity>
     }
 }
 
+void Manager::restoreHardwaresStatusEvent()
+{
+    clearHardwaresStatusEvent();
+
+    std::for_each(
+        _requiredHwsPdbgClass.begin(), _requiredHwsPdbgClass.end(),
+        [this](const auto& ele) {
+            struct pdbg_target* tgt;
+            pdbg_for_each_class_target(ele.c_str(), tgt)
+            {
+                try
+                {
+                    ATTR_HWAS_STATE_Type hwasState;
+                    if (DT_GET_PROP(ATTR_HWAS_STATE, tgt, hwasState))
+                    {
+                        log<level::ERR>(
+                            fmt::format("Skipping to create the hardware "
```

Comment:
This is an error scenario , should be creating error log here.

Reply:
I think I need some more discussion to add the Error log (aka PEL) because you added that kind of comment in many places/other commits as well.

I guess, you mean it as `commit/report` right?

I like to address based on the needs at the different places.